src/main/java/org/reactome/addlinks/dataretrieval/ensembl/EnsemblBioMartRetriever.java

```diff
@@ -98,26 +105,20 @@ private String getMicroarrayTypesQuery(String bioMartSpeciesName) {
      * @param biomartSpeciesName - String of species name in BioMart format (eg: hsapiens)
      * @param biomartQueryFilePath - String, Path to the BioMart XML Query file
      * @param biomartDataType - String, Type of data (either UniProt or Microarray) that will be queried for from BioMart
-     * @param fileSuffix - String that will be used to make filename (either _uniprot or _microarray) that will hold associated data.
+     * @param biomartFilename - Name of file (ending either with _uniprot or _microarray_go_ncbi_ids) that will hold associated data.
      * @throws IOException - Thrown when unable to write data to file.
      */
-    private void queryBioMartAndStoreData(String biomartSpeciesName, String biomartQueryFilePath, String biomartDataType, String fileSuffix) throws IOException {
-        String biomartFilename = this.destination + biomartSpeciesName + fileSuffix;
+    private void queryBioMartAndStoreData(String biomartSpeciesName, String biomartQueryFilePath, String biomartDataType, String biomartFilename) throws IOException {
         logger.info("Querying Biomart for species: {}; data type: {}", biomartSpeciesName, biomartDataType);
-        if (!Files.exists(Paths.get(biomartFilename))) {
-            Set<String> biomartResponseLines = new HashSet<>();
-            logger.info("Retrieving data associated with query ID: {}", biomartDataType);
-            try {
-                biomartResponseLines = queryBioMart(getBioMartIdentifierQuery(biomartQueryFilePath, biomartSpeciesName, biomartDataType), biomartDataType);
-            } catch (Exception e) {
-                logger.error("Unable to retrieve data associated with query ID: " + biomartDataType, e);
-                e.printStackTrace();
-            }
-            storeBioMartData(biomartFilename, biomartResponseLines);
-        }
-        else {
-        	logger.info("{} already exists. It is *assumed* to be complete, and will not be downloaded.", biomartFilename);
+        Set<String> biomartResponseLines = new HashSet<>();
+        logger.info("Retrieving data associated with query ID: {}", biomartDataType);
+        try {
+            biomartResponseLines = queryBioMart(getBioMartIdentifierQuery(biomartQueryFilePath, biomartSpeciesName, biomartDataType), biomartDataType);
```

Comment:
Since this is a method call using the result of another method call, could this be broken up on a few lines and indented so it's clearer at a glance which method call is getting which parameters?